packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -628,15 +660,23 @@ abstract class DiagnosticsNode {
 
   /// Whether the diagnostics should be hidden when showing the default
   /// view of a tree.
+  ///
+  /// This could be set to true (hiding the property) if another property
+  /// provides a normally more useful summary of the value.
```

Comment:
This comment is for DiagnosticNode which isn't always a property so the comment shouldn't talk about properties.